src/rpc_api/data_types.rs

```diff
@@ -407,6 +408,28 @@ pub struct MinerPowerLotusJson {
     has_min_power: bool,
 }
 
+// Note: kept the name in line with Lotus implementation for cross-referencing simplicity.
+#[derive(Clone, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "PascalCase")]
+pub struct MiningBaseInfo {
+    #[serde(with = "crate::lotus_json")]
+    pub miner_power: fvm_shared2::sector::StoragePower,
```

Comment:
Let's go through our shim.
```suggestion
    pub miner_power: crate::shim::sector::StoragePower,
```